py/picca/data.py

```diff
@@ -1126,7 +1130,11 @@ def from_fitsio(cls, hdu, pk1d_type=False):
             mjd=los_id
             fiberid=los_id
         else:
-            raise Exception("Could not find THING_ID or LOS_ID")
+            los_id = -1
```

Comment:
Why are we doing this? `los_id` is used to identify the different lines of sight.  I think this is a good way to check whether we are reading a properly formatted set of deltas. However, I could be convinced to remove this if there is a good reason to do so

Reply:
I think this was in order to read Naim's delta files (from pixsim).
Agree it's not a good idea to keep this here, I guess we should remove it and write a wrapper to "unproperly formatted" deltas.

Reply:
Reversed changes from yesterday's discussion